packages/sanity/src/core/form/inputs/PortableText/object/BlockObject.tsx

```diff
@@ -428,7 +435,7 @@ export const DefaultBlockObjectComponent = (props: BlockProps) => {
           value,
         })}
       </Root>
-      {open && (
+      {open && !nestedObjectNavigationEnabled && (
```

Comment:
In situations where we are using the new nested method, we do not want to show this object edit modal. However, in cases where we aren't, the old modal needs to work as expected